test/integration/targets/vault/runme.sh

```diff
@@ -2,6 +2,45 @@
 
 set -eux
 
+MYTMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
+trap 'rm -rf "${MYTMPDIR}"' EXIT
+
+# create a test file
+TEST_FILE="${MYTMPDIR}/test_file"
+echo "This is a test file" > "${TEST_FILE}"
+
+
+# encrypt it
+ansible-vault encrypt --vault-password-file vault-password "${TEST_FILE}"
```

Comment:
I see the `"$@"` added to the vault commands later in the script, but not this one or the two below it. Is that intentional?